addons/mail/static/src/components/discuss_mobile_mailbox_selection/discuss_mobile_mailbox_selection_tests.js

```diff
@@ -91,6 +92,46 @@ QUnit.test('select another mailbox', async function (assert) {
     );
 });
 
+QUnit.test('Navigate on mobile tabs', async function (assert) {
```

Comment:
This test should be split in 2 (maybe):
- 1 for open discuss from channel id in active id to correctly set correct tab
- 1 for tab change on click (isn't what the test above does??? Not sure it's relevant)

Ideally test have very few assertions, like a single one, so it makes sense to name test like an assertion comment. Generic name tends to either define big qunit tests or obscure tests, which is very very bad.

Reply:
It's relevant for following the case:

On mobile when you come from a notification on discuss, it's open the thread and set the `channel_type` linked as the discuss selected tabs. After that, if you navigate to the `mailboxes` tab the inner tab `inbox` isn't selected and you see [this](https://photos.app.goo.gl/onoKRXGMbTcEvFAv6).

The current test simulates this behavior, we load discuss with `channel` tab and we go to `mailboxes` tab. As you can see if you remove this [block](https://github.com/odoo-dev/odoo/blob/7eacaec911c2d2086c54961960374c6459f88d67/addons/mail/static/src/components/discuss/discuss.js#L274-L276) this test fails.